Localization/frFR.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+	Bagnon Localization Information: English Language
+		This file must be present to have partial translations
+--]] 
+
+local L = LibStub('AceLocale-3.0'):NewLocale('Groupie', 'frFR')
+
+if L then
```

Comment:
Simplify indentation

```suggestion
if not L then return end
```